cmd/earthly/registry_cmds.go

```diff
@@ -55,50 +63,43 @@ func (app *earthlyApp) registryCmds() []*cli.Command {
 					Required:    false,
 					Destination: &app.registryPasswordStdin,
 				},
-			},
-		},
-		{
-			Name:  "list",
-			Usage: "List configured registries *beta*",
-			UsageText: "earthly registry list\n" +
-				"	earthly registry list --org <org> --project <project>\n",
-			Action: app.actionRegistryList,
-			Flags: []cli.Flag{
-				&cli.StringFlag{
-					Name:        "org",
-					Usage:       "The organization to which the project belongs.",
-					Required:    false,
-					Destination: &app.orgName,
-				},
 				&cli.StringFlag{
-					Name:        "project",
-					Usage:       "The organization project in which to store registry credentials, if empty credentials will be stored under the user's secret storage.",
+					Name:        "aws-access-key-id",
+					EnvVars:     []string{"AWS_ACCESS_KEY_ID"},
+					Usage:       "AWS access key ID to use for elastic-container-registry.",
 					Required:    false,
-					Destination: &app.projectName,
+					Destination: &app.awsAccessKeyID,
 				},
-			},
-		},
-		{
-			Name:  "logout",
-			Usage: "Logout of a registry (that has credentials stored in earthly-cloud) *beta*",
-			UsageText: "earthly registry logout [<host>]\n" +
-				"	earthly registry login --org <org> --project <project> [<host>]\n",
-			Action: app.actionRegistryLogout,
-			Flags: []cli.Flag{
 				&cli.StringFlag{
-					Name:        "org",
-					Usage:       "The organization to which the project belongs.",
+					Name:        "aws-secret-access-key",
+					EnvVars:     []string{"AWS_SECRET_ACCESS_KEY"},
+					Usage:       "AWS secret access key to use for elastic-container-registry.",
 					Required:    false,
-					Destination: &app.orgName,
+					Destination: &app.awsSecretAccessKey,
 				},
 				&cli.StringFlag{
-					Name:        "project",
-					Usage:       "The organization project in which to store registry credentials, if empty credentials will be stored under the user's secret storage.",
+					Name:        "gcp-key",
+					EnvVars:     []string{"GCP_KEY"},
+					Usage:       "GCP key to use for artifact or container registry.",
```

Comment:
perhaps it would be good to clarify the key is a service account json or maybe even rename it.
This link might offer some good options - https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating